src/foam/nanos/auth/User.js

```diff
@@ -229,36 +228,47 @@ foam.CLASS({
       section: 'administrative'
     },
     {
-      class: 'FObjectProperty',
-      of: 'foam.nanos.auth.Phone',
-      name: 'phone',
+      class: 'String',
+      name: 'phoneNumber',
       documentation: 'Personal phone number.',
-      factory: function() {
-        return this.Phone.create();
-      },
-      view: { class: 'foam.u2.detail.VerticalDetailView' },
+//      validationPredicates: [
+//        {
+//          args: ['this'],
+//          predicateFactory: function(e) {
+//            return e.REG_EXP(
+//              e,
+//              /^(?:\+?1[-.●]?)?\(?([0-9]{3})\)?[-.●]?([0-9]{3})[-.●]?([0-9]{4})$/);
+//          },
+//          errorString: 'Please enter a phone number'
+//        }
+//      ],
       section: 'personal'
     },
     {
-      class: 'String',
-      name: 'phoneNumber',
-      transient: true,
-      documentation: `Omits properties of the phone number object and returns
-        the phone number.`,
-      expression: function(phone) {
-        return phone.number;
-      },
+      class: 'Boolean',
+      name: 'phoneNumberVerified',
```

Comment:
We don't want users to be able to set this value.

```suggestion
      name: 'phoneNumberVerified',
      writePermissionRequired: true
```

Reply:
In some files (user.js, contact.js, business.js) you can find:
    {
      class: 'Boolean',
      name: 'emailVerified',
      hidden: true
    }
I think maybe your remark applies here also. No?!

Reply:
You're right, it does. We need to do a survey/review at some point and make sure we're doing that wherever it makes sense to.